### PR TITLE
Add PostgreSQL database to infrastructure code

### PR DESCRIPTION
**Pull request by lgonczar**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/RPE-268

### Change description ###

Add PostgreSQL database to infrastructure code

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: infrastructure/variables.tf [hunk 1]
```diff
@@ -8,6 +8,11 @@ variable "location" {
   default = "UK South"
 }
```
Comment (rkondratowicz) [inline infrastructure/variables.tf]: worth renaming this to `location_app` (or similar) now.

Comment (lgonczar) [reply infrastructure/variables.tf]: Right. `location` and `location_db` is confusing. Will change.

File: infrastructure/main.tf [hunk 2]
```diff
@@ -35,16 +35,30 @@ resource "vault_generic_secret" "servicebus-listen-conn-string" {
...
 module "send-letter-producer-service" {
   source              = "git@github.com:contino/moj-module-webapp?ref=master"
   product             = "${var.product}-producer"
-  location            = "${var.location}"
+  location            = "${var.location_app}"
   env                 = "${var.env}"
   ilbIp               = "${var.ilbIp}"
   resource_group_name = "${azurerm_resource_group.rg.name}"
 
   app_settings = {
```
Comment (nitinprabhu) [inline infrastructure/main.tf]: very minor one formatting of the all the variables in this block could be consistent

Comment (lgonczar) [reply infrastructure/main.tf]: Did you mean  `=` alignment?

**Pull request merged**